package.json

```diff
@@ -63,6 +63,7 @@
     "lodash.isequal": "^4.5.0",
     "material-ui-color-picker": "^3.5.1",
     "neo4j-client-sso": "^1.2.2",
+    "@neo4j_solutions/solutions-auth-lib": "^1.0.1",
```

Comment:
Thinking about the future of including this branch into the open-source project.
Can we make it so that the app will still build if we don't have access to this package in the private solutions npm registry? How would we design this?